### PR TITLE
Kanvi/Changes for using the complete path for include files

### PR DESCRIPTION
**Pull request by kanvi-nervana**
1. changes name from `src` -> `ngraph_bridge`
2. #include "ngraph_utils.h" --> #include "ngraph_bridge/ngraph_utils.h"
3. order of header includes: stl/tensorflow/ngraph/ngraph-bridge

### CODE REVIEW

File: ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc [hunk 1]
```diff
@@ -17,10 +17,10 @@
 #include "tensorflow/core/graph/graph.h"
 #include "tensorflow/core/graph/node_builder.h"
 
-#include "ngraph_api.h"
-#include "ngraph_capture_variables.h"
+#include "ngraph_bridge/ngraph_api.h"
+#include "ngraph_bridge/ngraph_capture_variables.h"
+#include "ngraph_bridge/ngraph_utils.h"
 #include "ngraph_replace_op_utilities.h"
```
Comment (avijit-nervana) [inline ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc]: Missed this one?

File: ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc [hunk 2]
```diff
@@ -26,10 +26,10 @@
 
 #include "ngraph/event_tracing.hpp"
 #include "ngraph/runtime/backend.hpp"
+#include "ngraph_bridge/ngraph_freshness_tracker.h"
+#include "ngraph_bridge/ngraph_timer.h"
+#include "ngraph_bridge/ngraph_utils.h"
 #include "ngraph_catalog.h"
```
Comment (avijit-nervana) [inline ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc]: Missed this one?

File: ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc (hunk 1 above)
Comment (kanvi-nervana) [reply ngraph_bridge/enable_variable_ops/ngraph_capture_variables.cc]: `ngraph_replace_op_utilities.h` and `ngraph_cpature_variables.cc` are in the same folder

File: ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc (hunk 2 above)
Comment (kanvi-nervana) [reply ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc]: `ngraph_catalog.h` and `ngraph_catalog.h` are in the same folder

File: bazel/BUILD [hunk 3]
```diff
@@ -52,27 +52,27 @@ cc_library(
...
+        "ngraph_bridge/ngraph_backend_manager.cc",
+        "ngraph_bridge/ngraph_capture_variables.cc",
+        "ngraph_bridge/ngraph_cluster_manager.cc",
+        "ngraph_bridge/ngraph_deassign_clusters.cc",
+        "ngraph_bridge/ngraph_encapsulate_clusters.cc",
+        "ngraph_bridge/ngraph_encapsulate_op.cc",
+        "ngraph_bridge/ngraph_freshness_tracker.cc",
+        "ngraph_bridge/ngraph_mark_for_clustering.cc",
+        "ngraph_bridge/ngraph_rewrite_for_tracking.cc",
+        #"ngraph_bridge/ngraph_rewrite_pass.cc",
```
Comment (kanvi-nervana) [inline bazel/BUILD]:
```suggestion
```

File: ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc (hunk 2 above)
Comment (avijit-nervana) [reply ngraph_bridge/enable_variable_ops/ngraph_assign_op.cc]: Yes - need to add complete paths even if they are in the same folder. Please see examples from TensorFlow source code.

File: ngraph_bridge/ngraph_mark_for_clustering.cc [hunk 4]
```diff
@@ -13,12 +13,15 @@
...
-#include "ngraph_utils.h"
-#include "ngraph_version_utils.h"
+
 #include "tensorflow/core/graph/graph.h"
+
+#include "ngraph_bridge/ngraph_api.h"  // TODO: Importing this first causes a compile error
+#include "ngraph_bridge/ngraph_backend_manager.h"
+#include "ngraph_bridge/ngraph_mark_for_clustering.h"
+#include "ngraph_bridge/ngraph_utils.h"
+#include "ngraph_bridge/ngraph_version_utils.h"
```
Comment (sayantan-nervana) [inline ngraph_bridge/ngraph_mark_for_clustering.cc]: maybe this `TODO` is not relevant right now, but it does recommend importing `ngraph_mark_for_clustering` before `ngraph_api` (but i forget why)

Comment (avijit-nervana) [reply ngraph_bridge/ngraph_mark_for_clustering.cc]: Need to fix this - the need for of include files indicate that there would be a problem in future. Need to fix this.

Comment (kanvi-nervana) [reply ngraph_bridge/ngraph_mark_for_clustering.cc]: We are importing it, before `ngraph_mark_for_clustering` and the compiler does not complain. I will remove the comment.

**Pull request merged**